### PR TITLE
Manual.md: document the strange case of glib-devel and libglib-devel

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: Manual.md [hunk 1]
```diff
@@ -1731,6 +1731,10 @@ the installed files (`./xbps-src show-files pkg`) before pushing new updates.
 - Make sure that binaries are not stripped by the software, let xbps-src do this;
 otherwise the `debug` packages won't have debugging symbols.
 
+- Almost all the times your package needs `glib` the makedepends should require
```
Comment (Cogitri) [inline Manual.md]: > If your package depends on `glib` you should add `libglib-devel` to `makedepends`. Only add `glib-devel` to `hostmakedepends` if one the the tools/(maybe binaries instead?) (e.g. glib-compile-schemas) are required for the build.

That sounds a little better IMHO

File: Manual.md [hunk 2]
```diff
@@ -1731,6 +1731,10 @@ the installed files (`./xbps-src show-files pkg`) before pushing new updates.
 - Make sure that binaries are not stripped by the software, let xbps-src do this;
 otherwise the `debug` packages won't have debugging symbols.
 
+- If your package needs `glib` then `libglib-devel` should be added to makedepends.
```
Comment (Vaelatern) [inline Manual.md]:
```
If your package needs `glib`, then
```

File: Manual.md [hunk 3]
```diff
@@ -1731,6 +1731,10 @@ the installed files (`./xbps-src show-files pkg`) before pushing new updates.
...
+Only add `glib-devel` to `hostmakedepends` if one of it's tools (e.g. glib-compile-schemas)
```
Comment (Vaelatern) [inline Manual.md]: `if one of its tools`

**Comment by maxice8** [comment]
On Sat, Oct 06, 2018 at 08:36:58PM -0700, Toyam Cox wrote:
> Vaelatern commented on this pull request.
> 
> 
> 
> > @@ -1731,6 +1731,10 @@ the installed files (`./xbps-src show-files pkg`) before pushing new updates.
>  - Make sure that binaries are not stripped by the software, let xbps-src do this;
>  otherwise the `debug` packages won't have debugging symbols.
>  
> +- If your package needs `glib` then `libglib-devel` should be added to makedepends.
> +Only add `glib-devel` to `hostmakedepends` if one of it's tools (e.g. glib-compile-schemas)
> 
> `if one of its tools`
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/3503#pullrequestreview-162276542

fixed, thanks

**Comment by Duncaen** [comment]
Hm not sure if `Manual.md` is the right place for package specific informations.

**Comment by maxice8** [comment]
We need to put it somewhere and link to it. What you suggest?

**Comment by Duncaen** [comment]
Don't know, maybe a new `FAQ.md` because this would as far as I know the first specific package mentioned in `Manual.md`.

**Comment by Cogitri** [comment]
IMHO it makes sense to put it in Manual.md. I don't think that we will have to document (much) more package specific stuff and I'd already be happy if more people actually read `Manual.md`.

**Comment by Chocimier** [comment]
Why is peculiar glib-devel documented to use different than others *-devel instead of renamed to something not confusing like glib-tools?

**Comment by maxice8** [comment]
On Tue, Oct 09, 2018 at 03:48:41AM -0700, Piotr wrote:
> Why is peculiar glib-devel documented to use different than others *-devel instead of renamed to something not confusing like glib-tools?
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/3503#issuecomment-428146868

It has always been this way since 2009 (85cc462e1d738c085d9f21f8d24566e1a22a2fbf) i don't think you're getting an answer as to why from xtr tho.

**Pull request closed without merging**